tests/test_parameter.py

```diff
@@ -149,9 +149,8 @@ def test_density_matrix_xreplace() -> None:
     assert np.allclose(dm.xreplace({alpha: 1, beta: 2}).rho, np.array([[1, 2], [1, 2]]))
 
 
-@pytest.mark.parametrize("jumps", range(1, 11))
 @pytest.mark.parametrize("use_xreplace", [False, True])
-def test_random_circuit_with_parameters(fx_rng: Generator, jumps: int, use_xreplace: bool) -> None:
+def test_random_circuit_with_parameters(fx_rng: Generator, use_xreplace: bool) -> None:
```

Comment:
The intention was to use `fx_bg` instead of `fx_rng`.
```suggestion
def test_random_circuit_with_parameters(fx_bg: PCG64, jumps: int, use_xreplace: bool) -> None:
    rng = Generator(fx_bg.jumped(jumps))
```

Reply:
Fixed in edcdac27d261dc2a6a541e8f71771e9ef14ac9bc .